public/core.d.ts

```diff
@@ -1016,6 +1016,541 @@ declare namespace JMap {
+       * 
+       * // return true
+       * JMap.Layer.DynamicLayer.isConditionValueValid("EQUALS", "string", "")
+       *  
+       * // return false
+       * JMap.Layer.DynamicLayer.isConditionValueValid("EQUALS", "string", undefined)
+       * 
+       * // return true
+       * JMap.Layer.DynamicLayer.isConditionValueValid("IS_EMPTY", "string")
+       * 
+       * // return false
+       * JMap.Layer.DynamicLayer.isConditionValueValid("EQUALS", "number", "test")
+       * 
+       * // return true
+       * JMap.Layer.DynamicLayer.isConditionValueValid("EQUALS", "number", 22)
+       * ```
+       */
+      function isConditionValueValid(operator: JDynamicFilterOperator, attributeType: JLayerAttributeType, value1?: any, value2?: any): boolean
+
+      /**
+       * **JMap.Layer.DynamicLayer.isAttributeTypeAcceptMultipleValueOperators**
+       * 
+       * Returns true if the attribute type accept multiple value operator ("EQUALS" or "NOT_EQUALS" for example).
+       * 
+       * This function is safe, it throws nothing.
+       * 
+       * @param attributeType The attribute type
+       * @example ```ts
+       * 
+       * // return true
+       * JMap.Layer.DynamicLayer.isAttributeTypeAcceptMultipleValueOperators("string")
+       *  
+       * // return true
+       * JMap.Layer.DynamicLayer.isAttributeTypeAcceptMultipleValueOperators("number")
+       * 
+       * // return false
+       * JMap.Layer.DynamicLayer.isAttributeTypeAcceptMultipleValueOperators("date")
+       * 
+       * // return false
+       * JMap.Layer.DynamicLayer.isAttributeTypeAcceptMultipleValueOperators("datetime")
+       *  
+       * // return false
+       * JMap.Layer.DynamicLayer.isAttributeTypeAcceptMultipleValueOperators("boolean")
+       * ```
+       */
+      function isAttributeTypeAcceptMultipleValueOperators(attributeType: JLayerAttributeType): boolean
+
+      /**
+       * **JMap.Layer.DynamicLayer.isAttributeTypeAcceptTwoValuesOperators**
+       * 
+       * Returns true if the attribute type accept two value operator ("IS_IN_RANGE").
+       * 
+       * This function is safe, it throws nothing.
+       * 
+       * @param attributeType The attribute type
+       * @example ```ts
+       * 
+       * // return false
+       * JMap.Layer.DynamicLayer.isAttributeTypeAcceptTwoValuesOperators("string")
+       * 
+       * // return true
+       * JMap.Layer.DynamicLayer.isAttributeTypeAcceptTwoValuesOperators("date")
+       * 
+       * // return true
+       * JMap.Layer.DynamicLayer.isAttributeTypeAcceptTwoValuesOperators("datetime")
+       *  
+       * // return true
+       * JMap.Layer.DynamicLayer.isAttributeTypeAcceptTwoValuesOperators("number")
+       *  
+       * // return false
+       * JMap.Layer.DynamicLayer.isAttributeTypeAcceptTwoValuesOperators("boolean")
+       * ```
+       */
+      function isAttributeTypeAcceptTwoValuesOperators(attributeType: JLayerAttributeType): boolean
```

Comment:
Ici en théorie on devrait dire `doesAttributeTypeAcceptTwoValuesOperators`

Reply:
En fait une fonction qui retourne un boolean c'est bien de la faire commencer par is ou has je trouve.

Reply:
Mais "Accept" c'est un verbe, `is, has` c'est pour des propriétés. "does" aussi c'est un préfixe de type boolean, comme:

- does
- can